cabal.project

```diff
@@ -0,0 +1,6 @@
+packages: .
```

Comment:
Is this necessary? Or helpful for other reasons?

Reply:
It's necessary if you have a `cabal.project` at all (otherwise it complains that there is nothing to do).